business-registry-dashboard/app/components/Form/AddBusiness/Base.vue

```diff
@@ -125,7 +125,8 @@ async function handleEmailOption () {
       fromOrgId: Number(accountStore.currentAccount.id),
       businessIdentifier: props.identifier,
       // @ts-expect-error - toOrgId has to be null, as this is a bug on the backend
-      toOrgId: null
+      toOrgId: null,
+      type: 'EMAIL'
```

Comment:
How does this fix the issue?

Reply:
Never mind. I see in handleDelegationOption() below that the type is something else.

Reply:
currently in the API we default this one to email, I'd rather explicitly set it 

That way we can do an auth check for delegation and auth check for magic link (two different checks)